docs/lang/array.md

```diff
@@ -67,23 +67,22 @@ int main() {
 }
 ```
 
-### 越界访问下标
+### Out-of-bound access to index
 
-数组的下标 $\mathit{idx}$ 应当满足 $0\leq \mathit{idx}< \mathit{size}$ ，如果下标越界，则会产生不可预料的后果，如段错误（Segmentation Fault），或者修改预期以外的变量。
+The index $\mathit{idx}$ of the array should satisfy $0\leq \mathit{idx}< \mathit{size}$ . If the index goes out of range, unpredictable result will occur, such as segmentation fault, or modifying variables other than expected ones.
```

Comment:
result occur

there may be unpredictable consequences, including causing seg fault and modifing ...